test/unit/test_autosubmit_helper.py

```diff
@@ -71,3 +70,44 @@ def test_get_allowed_members(as_member_list, ids, return_list_value, result,
     as_member_list.return_value = return_list_value
 
     assert helper.get_allowed_members(ids, as_config) == result
+
+
+@patch('autosubmit.helpers.autosubmit_helper.sleep')
+@patch('autosubmit.helpers.autosubmit_helper.ExperimentHistory')
+@patch('autosubmit.helpers.autosubmit_helper.check_experiment_exists')
+@pytest.mark.parametrize('time, header_skip, experiment_exists', [
+    ('a000', False, False),
+    ('04-00-00', False, False),
+    ('04-00-00', False, True),
+    ('04:00:00', True, False),
+    ('04:00:00', True, True),
+], ids=['expid instead of time', 'wrong format hours', 'right format hours', 'fulldate wrong format',
+        'fulldate wrong format false'])
+def teste_handle_start_after(autosubmit_helper, mock_experiment_history,
+                             mocked_sleep, autosubmit_config: Callable, time: str,
+                             header_skip: bool, experiment_exists: bool):
+    """
+    function to test the function handle_start_time inside autosubmit_helper
+    """
+    expid = 'a000'
+    autosubmit_config(expid, experiment_data={})
+
+    experiment_history = experiment_history2 = MagicMock(spec='experiment_history')
```

Comment:
This is creating a single mock object. Both instances are pointing to the same object.

In general, multiple assignments in a single line is not very common. I myself have trouble when I read code with that as I sometimes skip the assignments.